requirements/base.txt

```diff
@@ -1,13 +1,8 @@
--r multidict.txt
--r typing-extensions.txt
-# required c-ares will not build on windows
-aiodns==3.0.0; sys_platform=="linux" or sys_platform=="darwin"
-aiosignal==1.3.1
-async-timeout==4.0.2
-Brotli==1.0.9
-cchardet==2.1.7; python_version < "3.10"  # Unmaintained: aio-libs/aiohttp#6819
-charset-normalizer==3.2.0
-frozenlist==1.4.0
-gunicorn==21.2.0
-uvloop==0.17.0; platform_system!="Windows" and implementation_name=="cpython" and python_version<"3.9" # MagicStack/uvloop#14
-yarl==1.9.2
+#
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
+#
+#    pip-compile --allow-unsafe --output-file=requirements/base.txt --strip-extras requirements/multidict.txt
```

Comment:
Looks like I messed up arguments of the for-loop I used locally. Will have to hotfix the lock files in a follow-up PR.